src/components/LogoutButton/LogoutButtonComponent.js

```diff
@@ -1,15 +1,19 @@
-import React, { Component } from 'react';
-
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
+import { Typography } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  button: {
+    textTransform: 'capitalize',
```

Comment:
I don't think we need this style. We have Logout starting with a capital letter already in the code:
```
      <Typography variant="subtitle1">Logout</Typography>
```
So I believe we don't need to capitalize it. It's already capitazlied.